core/ts/MaciState.ts

```diff
@@ -1563,34 +1559,13 @@ const genSubsidyVkSig = (
     _stateTreeDepth: number,
     _intStateTreeDepth: number,
     _voteOptionTreeDepth: number,
-): BigInt => {
+): bigint => {
     return (BigInt(_stateTreeDepth) << BigInt(128)) +
            (BigInt(_intStateTreeDepth) << BigInt(64)) +
             BigInt(_voteOptionTreeDepth)
 }
 
 
-/*
- * A helper function which hashes a list of results with a salt and returns the
- * hash.
- *
- * @param results A list of vote weights
```

Comment:
Think we should retain this description somewhere.  
It's currently hard to understand what 'tallyFileData.perVOSpentVoiceCredits.tally.map((x) => BigInt(x))' means like following function call:
```
genTreeCommitment(
        tallyFileData.perVOSpentVoiceCredits.tally.map((x) => BigInt(x)),
        BigInt(tallyFileData.perVOSpentVoiceCredits.salt),
        poll.treeDepths.voteOptionTreeDepth)
```

Reply:
isn't the same (aside changing the word results to leaves) description on the genTreeCommitment function inside crypto/ts/utils.ts?

Reply:
Not sure if someone could figure out `tallyFileData.perVOSpentVoiceCredits.tally.map((x) => BigInt(x)` means a lists of 'vote weights'. 
Think we can separate the issue of 'documenting how to compute  `newResultsCommitment`, `newSpentVoiceCreditsCommitment`...etc in: https://github.com/privacy-scaling-explorations/maci/blob/c1b8b75698ed920f755fa1765e1a168096655fb3/cli/ts/genProofs.ts#L547-L557